<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifierWidget</class>
 <widget class="QWidget" name="ClassifierWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="gaussianButton">
     <property name="text">
      <string>Gaussian</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="gaussianThresholdButton">
     <property name="text">
      <string>Gaussian + Threshold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="boundingBoxButton">
     <property name="text">
      <string>Bounding Box</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="scalingLabel">
        <property name="text">
         <string>Scaling:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="thresholdLabel">
        <property name="text">
         <string>Threshold:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="scalingWidget" native="true">
        <layout class="QHBoxLayout" name="scalingLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="scalingSlider">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SkinCam::NumberLabel" name="scalingNumber">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="thresholdWidget" native="true">
        <layout class="QHBoxLayout" name="thresholdLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="thresholdSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SkinCam::NumberLabel" name="thresholdNumber">
           <property name="text">
            <string>10 %</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkinCam::NumberLabel</class>
   <extends>QLabel</extends>
   <header>numberlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
